packages/toolpad-core/src/DashboardLayout/DashboardLayout.tsx

```diff
@@ -286,7 +291,7 @@ function DashboardLayout(props: DashboardLayoutProps) {
 
   const getDrawerSharedSx = React.useCallback(
```

Comment:
In this `getDrawerSharedSx` callback, `sidebarExpandedWidth` also needs to be added at the end to the dependencies array.
The `react-hooks/exhaustive-deps` eslint rule should automatically show you an error about this.

Reply:
Hi @apedroferreira, changes are made accordingly, please double check.